test/python/fx_importer/basic_test.py

```diff
@@ -13,26 +13,7 @@
 import torch.export
 import torch.nn as nn
 
```

Comment:
This was pre-existing, but the copyright notice in this file seems inconsistent with the rest of the project (AMD line). Is that intentional, or an oversight?

Reply:
Not sure, but changed to be consistent.